pypeit/par/pypeitpar.py

```diff
@@ -852,8 +852,10 @@ def __init__(self, spec_method=None, spec_maxshift=None, spectrum=None,
 
         defaults['spectrum'] = 'paranal_sky.fits'
         dtypes['spectrum'] = str
-        descr['spectrum'] = 'Archive sky spectrum to be used for the flexure correction.'
-
+        descr['spectrum'] = ('Archive sky spectrum to be used for the flexure correction. '
+                             'See ``pypeit/data/sky_spec/`` for a list of available sky spectra. '
+                             'If ``model`` is used, a model sky spectrum will be generated '
```

Comment:
point out here too it is generated from the the `objsky` spectrum, i.e. its resolution, etc.